doc/source/user_guide/data_sources_tutorial.rst

```diff
@@ -0,0 +1,310 @@
+.. _user_guide_data_sources:
+
+===============
+DPF DataSources
+===============
+
+The DataSources operator is a class available as a submodule of the 'ansys.dpf.core'
+package. It contains files with the analysis results. So this object is used in
+the first basic step: define simulation data, by defining where it is located.
```

Comment:
```suggestion
package. It manages paths to files. Use this object to declare data inputs for DPF and define their locations.
```

Reply:
@luisaFelixSalles in English you should refrain from using the passive form. We try to follow the Google docs guidelines. At the end of the day, for PRs like this you will also have to add @JennaPaikowsky as a reviewer.